provider/resource_on_call_role.go

```diff
@@ -46,11 +46,8 @@ func resourceOnCallRole() *schema.Resource {
 
 			"system_role": &schema.Schema{
 				Type:        schema.TypeString,
-				Computed:    false,
-				Required:    true,
-				Optional:    false,
-				ForceNew:    false,
-				Description: "The kind of role",
+				Computed:    true,
```

Comment:
This file is code generated from Rootly's JSON-API schema. These changes will get wiped out with the next build, unless these changes were made to the JSON-API schema in lib/api/v1/schemas/on_call_roles_schema.rb

In the schema for creating a role you need to set this as an optional field (it's currently marked required on L34), and also add `tf_computed: true` in the object schema L86) to ensure it's also marked as a computed attribute. Then the next time the Terraform provider is built this code will be regenerated to match the JSON-API schema.